public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -1,37 +1,43 @@
 import React, { PureComponent } from 'react';
 import AutoSizer from 'react-virtualized-auto-sizer';
 import { saveAs } from 'file-saver';
-import { css } from 'emotion';
+import { css, cx } from 'emotion';
 
 import { InspectHeader } from './InspectHeader';
 
 import { DashboardModel, PanelModel } from 'app/features/dashboard/state';
 import {
-  JSONFormatter,
+  Button,
+  CustomScrollbar,
   Drawer,
+  JSONFormatter,
   LegacyForms,
-  Table,
-  TabContent,
   stylesFactory,
-  CustomScrollbar,
-  Button,
+  TabContent,
+  Table,
 } from '@grafana/ui';
-const { Select } = LegacyForms;
-import { getLocationSrv, getDataSourceSrv } from '@grafana/runtime';
+
+import { getDataSourceSrv, getLocationSrv } from '@grafana/runtime';
 import {
-  DataFrame,
-  DataSourceApi,
-  SelectableValue,
   applyFieldOverrides,
-  toCSV,
+  DataFrame,
   DataQueryError,
-  PanelData,
-  getValueFormat,
+  DataSourceApi,
+  DataTransformerID,
   formattedValueToString,
+  getValueFormat,
+  PanelData,
   QueryResultMetaStat,
+  SelectableValue,
+  toCSV,
+  transformDataFrame,
+  transformersRegistry,
 } from '@grafana/data';
+
 import { config } from 'app/core/config';
 
+const { Select } = LegacyForms;
```

Comment:
Are we using legacy components in the new PanelInspector?

Reply:
+1